core/templates/dev/head/exploration_editor/main_tab/StateResponses.js

```diff
@@ -284,20 +286,27 @@ oppia.controller('StateResponses', [
       trainingDataService.initializeTrainingData(
         explorationId, currentStateName);
     };
+
     $scope.suppressDefaultAnswerGroupWarnings = function() {
       var interactionId = $scope.getCurrentInteractionId();
+      var answerGroups = responsesService.getAnswerGroups();
+      // Use answerChoices to store answer choice text for comparison
+      var answerChoices = [];
+      var customizationArgs = stateCustomizationArgsService.savedMemento;
+      // Use handledAnswersArray to check that
+      // each answer choice has been covered
+      var handledAnswersArray = [];
```

Comment:
```
This array contains a list of booleans, one for each answer choice. Each boolean is true if the corresponding answer has been covered by at least one rule, and false otherwise.
```